audit/projects/kubernetes-public/services/enabled.txt

```diff
@@ -3,6 +3,7 @@ bigquery.googleapis.com              BigQuery API
 bigquery.googleapis.com              BigQuery API
 bigquerystorage.googleapis.com       BigQuery Storage API
 clouderrorreporting.googleapis.com   Error Reporting API
+cloudfunctions.googleapis.com        Cloud Functions API
```

Comment:
We'll need to setup auditing for the cloudfunctions config here, and any new entries. (I think there are still quite a few in the backlog).